docs/sources/architecture/thing-definition.md

```diff
@@ -154,4 +154,20 @@ The channel group type is defined on the same level as the thing types and chann
 </channel-group-type> 
 ```
 
-When a thing will be created for a thing type with channel groups, the channel UID will contain the group ID in the last segment divided by a hash (#). If an Item should be linked to channel within a group, the channel UID would be `binding:multiChannelSwitchActor:myDevice:switchActor1#switch` for the XML example before.
\ No newline at end of file
+When a thing will be created for a thing type with channel groups, the channel UID will contain the group ID in the last segment divided by a hash (#). If an Item should be linked to channel within a group, the channel UID would be `binding:multiChannelSwitchActor:myDevice:switchActor1#switch` for the XML example before.
+
+## Properties
+As a binding developer you will often have the demand to add meta data to your thing type that represent no configuration parameter. As meta data you can consider any information or detail of your thing that you would like to make available to the system. For example, such meta data can be thing information like the vendor or the model id. To define the corresponding information the thing type definition provides the possibility to specify so-called `properties`:
```

Comment:
> As a binding developer you will often have the demand to add meta data to your thing type that represent no configuration parameter.

No, I don't. I am implementing an API and I am expecting the API to clearly tell me what I am supposed to do and what I should not do. As I am not asked to add properties, I therefore have no intrinsic demand to add any.